### PR TITLE
[PIN-2282] Added name and description to tenant

### PR DESCRIPTION
**Pull request by bloodsky**
I added name and description to tenant model. This is my first PR, hooray! 🎉 💻

### CODE REVIEW

**Review by beetlecrunch — APPROVED** [review]
🚀

**Review by galales — COMMENTED** [review]
LGTM 🥌 
before merging let's wait for confirmation that the `description` field is actually necessary

**Comment by bloodsky** [comment]
`description` field has been removed as we decided.

**Review by TonioGela — APPROVED** [review]
LGTM, before merging we definitively need to address this API change in all the other services too.

**Review by galales — APPROVED** [review]

File: src/main/scala/it/pagopa/interop/tenantmanagement/model/persistence/serializer/v1/protobufUtils.scala
Comment (TonioGela) [inline src/main/scala/it/pagopa/interop/tenantmanagement/model/persistence/serializer/v1/protobufUtils.scala]: There's not much value in having it defined here as a variable, can you inline it?

**Review by TonioGela — APPROVED** [review]

**Pull request merged**
